frontend/src/routes/@components/Error/index.tsx

```
@@ -0,0 +1,39 @@
+import { Link } from '@tanstack/react-router';
+
+import Button from '@/components/Button';
+import { Flex } from '@/components/Flex';
+import { Text } from '@/components/Text';
+import { vars } from '@/theme/index.css';
+
+const Error = () => (
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)